diskmaker/controllers/lv/reconcile_test.go

```diff
@@ -253,7 +253,7 @@ func TestCreateSymLinkByDeviceName(t *testing.T) {
 
 	d, _ := getFakeDiskMaker(t, tmpSymLinkTargetDir, lv, sc)
 	d.fsInterface = FakeFileSystemInterface{}
-	diskLocation := DiskLocation{fakeDisk.Name(), "", fakeDisk.Name(), internal.BlockDevice{}}
+	diskLocation := DiskLocation{fakeDisk.Name(), "", fakeDisk.Name(), internal.BlockDevice{}, false}
```

Comment:
I think it would be useful to add a linux-only e2e test that actually provides a small few MB ext2 or fat filesystem and expects it to be wiped after `createSymlink`.

For example, this test calls `createTmpFile` to create a file that the test uses as a "device", it should be possible either to copy a small image of a filesystem there or even create an empty 1MiB file and call `mkfs.ext2` on that file + expect it to be wiped (i.e. `file -s` does not return `ext2`)